src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+
+  if (days >= 3 && days < 7) {
+    return (rentCost * days) - 20;
+  }
+
+  if (days >= 7) {
+    return (rentCost * days) - 50;
+  }
+
+  return (rentCost * days);
```

Comment:
Also the (rentCost * days) is a recurring operation, maybe you could, like the values, store it somewhere for more readability